### PR TITLE
Add configuration for EIC launch call to action

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/elifesciences/issues/issues/4769

### CODE REVIEW

File: pillar/journal-public.sls [hunk 1]
```diff
@@ -4,7 +4,7 @@ journal:
     submit_url: https://submit.elifesciences.org/
     feature_xpub: false
     calls_to_action:
-      - path: ^(/$|/about$|/for-the-press$|/for-the-press/5cc21558/)
+      - path: ^(/$|/about$|/about/aims-scope|/for-the-press$|/for-the-press/5cc21558/)
```
Comment (thewilkybarkid) [inline pillar/journal-public.sls]: I took https://github.com/elifesciences/issues/issues/4769#issuecomment-469655965 to mean `/about$|/about/`

File: pillar/environment-preview-public.sls [hunk 2]
```diff
@@ -4,3 +4,21 @@ elife:
...
+      - id: eisen2019
+        path: ^(/$|/about$|/about/|/for-the-press$|/for-the-press/5cc21558/)
+        time:
+        text: 'Hear more about Michael Eisen, our new Editor-in-Chief, and the latest developments at eLife'
+        button:
+            text: 'Sign up for eLife news'
+            path: https://crm.elifesciences.org/crm/elife-news
+        image:
+            alt: ''
+            uri: https://iiif.elifesciences.org/journal-cms/person%%2F2019-03%%2Fmike.jpg
```
Comment (davidcmoulton) [inline pillar/environment-preview-public.sls]: `Bad request`. Presumably because it's not live yet?

File: pillar/environment-preview-public.sls [hunk 3]
```diff
@@ -4,3 +4,21 @@ elife:
...
+            source:
+                mediaType: image/jpg
+                uri: https://iiif.elifesciences.org/journal-cms/person%%2F2019-03%%2Fmike.jpg/full/full/0/default.jpg
```
Comment (davidcmoulton) [inline pillar/environment-preview-public.sls]: `Bad request` see above.

File: pillar/environment-preview-public.sls (hunk 2 above)
Comment (thewilkybarkid) [reply pillar/environment-preview-public.sls]: There's a double `%` (escaping for Symfony, change it to one and it should work).

**Pull request merged**